config/bootstrap.php

```diff
@@ -33,11 +33,21 @@
  */
 require CORE_PATH . 'config' . DS . 'bootstrap.php';
 
+// You can remove this if you are confident that your PHP version is sufficient.
+if (version_compare(PHP_VERSION, '5.5.9') < 0) {
```

Comment:
Reading a file and decoding JSON on every request is sub-optimal. The duplication is warranted here, and its not like we are going to _lower_ the version requirement in the future.